cmd/katalyst-agent/app/options/eviction/memory_pressure_eviction.go

```diff
@@ -30,15 +30,23 @@ type MemoryPressureEvictionOptions struct {
 	RSSOveruseEvictionFilter     string
 	SystemPressureSyncPeriod     int
 	SystemPressureCoolDownPeriod int
+	WorkloadPath                 string
+	MemPressureSomeThreshold     int
+	MemPressureFullThreshold     int
+	MemPressureDuration          int
 }
 
 // NewMemoryPressureEvictionOptions returns a new MemoryPressureEvictionOptions
 func NewMemoryPressureEvictionOptions() *MemoryPressureEvictionOptions {
 	return &MemoryPressureEvictionOptions{
-		SystemPressureSyncPeriod: 30,
+		SystemPressureSyncPeriod: 9,
```

Comment:
@lubinszARM what is the reason for adjusting the default parameters？will it have an impact on other scenarios?

Reply:
The default detection rules for memory.psi for all online services are as follows: mem.psi > 4, and duration=15 seconds, it triggers reclaimed-eviction, and if it exceeds 45 seconds, it triggers the eviction for all services, including shared.
So, here, I need to  reduce the frequency of periodic detection.

Reply:
For your reference, facebook's detection rule is : mem.psi > 5 + duration=15 secondes.
https://github.com/facebookincubator/oomd/blob/main/docs/configuration.md